web/modules/custom/customhook/customhook.module

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * @file
+ * This file demonstrates  how to use hook function
```

Comment:
`demonstrates how` extra space is there in between.
No full stop at the end of the line.